ml-agents/mlagents/trainers/simple_env_manager.py

```diff
@@ -53,9 +53,9 @@ def _reset_env(
     @property
     def external_brains(self) -> Dict[BehaviorName, BrainParameters]:
         result = {}
-        for brain_name in self.env.get_behavior_names():
+        for brain_name in self.env.behavior_specs:
```

Comment:
```suggestion
        for behavior_name, behavior_spec in self.env.behavior_specs.items():
```
(and update line below)